test/src/utils/sugar.rs

```diff
@@ -0,0 +1,17 @@
+use crate::utils::mine::mine_until_bool;
+use crate::Node;
+
+pub fn out_ibd_mode(nodes: &[Node]) {
+    for node in nodes.iter() {
+        mine_until_bool(node, || node.get_tip_block_number() > 0)
+    }
+}
```

Comment:
There already have `exit_ibd_mode()` in `test/src/net.rs`, so why need another `out_ibd_mode()` function here?